config/locales/simple_form.oc.yml

```diff
@@ -12,6 +12,8 @@ oc:
         data: Fichièr CSV exportat d’una autra instància Mastodon
       sessions:
         otp: Picatz lo còdi d’autentificacion en dos temps (Two factor code) de vòstre mobil o utilizatz un de vòstres còdis de recuperacion.
+  user:
```

Comment:
Off the top of my head I don't even remember which level allowed_languages is supposed to go on. Should't it be just under labels.defaults?